lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -856,3 +856,131 @@ def test_streamplot():
     ax.streamplot(x, y, u, v, transform=ccrs.PlateCarree(),
                   density=(1.5, 2), color=mag, linewidth=2 * mag)
     return fig
+
+
+@pytest.mark.natural_earth
+@pytest.mark.mpl_image_compare(filename='annotate_mercator.png', style='mpl20')
+def test_annotate_backwardscompat():
+    """ Work around for annotate to work from
+    https://stackoverflow.com/questions/25416600/why-the-annotate-worked-unexpected-here-in-cartopy/25421922#25421922
+    - check that an mpl_transform is passed through without issue.
+    """
+    fig = plt.figure(figsize=(10, 5))
+    ax = fig.add_subplot(1, 1, 1, projection=ccrs.Mercator())
+
+    ax.set_extent([65, 125, 5, 40])
+    ax.coastlines()
+    ax.plot(116.4, 39.95, 'ob', transform=ccrs.PlateCarree())
```

Comment:
I don't think we want to test plot() lines, but rather the annotations. So, I think we can drop all the `plot` and `scatter` calls in the tests.

Reply:
yeah, good call.  Removed the calls that aren't annotate in the consolidated test, except coastlines so we know it's all located with correctly with respect to 1 other feature.